src/Corporation/ui/modals/ExportModal.tsx

```diff
@@ -102,18 +101,18 @@ export function ExportModal(props: IProps): React.ReactElement {
         <br />
         For example: setting the amount "(EINV-20)/10" would try to export all except 20 of the material.
       </Typography>
-      <Select onChange={onIndustryChange} value={industry}>
-        {corp.divisions
-          .filter((division: Industry) => isRelevantMaterial(props.mat.name, division))
-          .map((division: Industry) => (
-            <MenuItem key={division.name} value={division.name}>
-              {division.name}
+      <Select onChange={onIndustryChange} value={division.name}>
+        {[...corp.divisions.values()]
+          .filter((div) => isRelevantMaterial(props.mat.name, div))
```

Comment:
I'd call this something else. "div" is overloaded with HTML div, especially in a JSX context.

Reply:
Yeah, related to the other comment about "division" being used as a frequent variable name.